com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -53,6 +48,7 @@ static class Style
             public const string resolveAll = "Fix All";
             public const string resolveAllQuality = "Fix All Qualities";
             public const string resolveAllBuildTarget = "Fix All Platforms";
+            public const string fixAllOnNonHDRP = "Current Quality is not using High Definition render pipeline. If you attempt a Fix All, the current Quality will be changed to use it.";
```

Comment:
I would rephrase with
The active Quality Level is not using a High Definition Render Pipeline asset. If you attempt a Fix All, the Quality Level will be changed to use it.